packages/jsoncpp/jsoncpp.pacscript

```diff
@@ -0,0 +1,53 @@
+pkgname="jsoncpp"
+gives="libjsoncpp25"
+provides=('libjsoncpp-dev' 'libjsoncpp.so')
+conflicts=('libjsoncpp25' 'libjsoncpp-dev' 'libjsoncpp.so')
+pkgver="1.9.5"
+pkgdesc='C++ library for interacting with JSON'
+url='https://github.com/open-source-parsers/jsoncpp'
+arch=('x86_64')
+license=('MIT')
+makedepends=('doxygen' 'graphviz' 'python3' 'meson' 'ninja-build')
+source=("https://github.com/open-source-parsers/${pkgname}/archive/${pkgver}/${pkgname}-${pkgver}.tar.gz")
+sha512sums=('1d06e044759b1e1a4cc4960189dd7e001a0a4389d7239a6d59295af995a553518e4e0337b4b4b817e70da5d9731a4c98655af90791b6287870b5ff8d73ad8873')
+maintainer=("xdavius <xdavius@github.com>")
+
+prepare() {
+  cd "${pkgname}-${pkgver}"
+  echo "${pkgver}" > version
+}
+
+build() {
+  cd "${pkgname}-${pkgver}"
+  echo "Building shared library..."
+  meson --prefix=/usr/local --default-library shared build
```

Comment:
Why not regular /usr?

Reply:
> Why not regular /usr?

https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1076787

Because in Debian/Ubuntu some packages are packaged in non standard way to avoid problems. 
We should install pacstall libs in /usr/local to avoid conflicts with systemlibs. Seems to be a better practice.

I will set all my future lib packages in /usr/local. Normaly if no prefix is defined, it should be the default path, /usr prefix is reserved to system libs, not foreign libs.